SubTemplates/IoT/template.yaml

```diff
@@ -69,15 +71,112 @@ Resources:
       Timeout: 10
       Policies:
         - AWSLambdaBasicExecutionRole
-  
+      Environment:
+        Variables:
+          ENV: dev
+      VpcConfig:
```

Comment:
This vpc config is blank and is causing errors for deploy

Reply:
This was added in my testing branch but didn't make it in the official checkin. There now.